home.html

```diff
@@ -0,0 +1,224 @@
+              </div>
+              <div class="program-title">
+                <h2>Rap Music</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-drum"></i>
+              </div>
+              <div class="program-title">
+                <h2>Live Show</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-guitar"></i>
+              </div>
+              <div class="program-title">
+                <h2>R&B Music</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+
+            <div class="program-box1">
+              <div class="top-icon">
+                <i class="fa-solid fa-3x fa-music"></i>
+              </div>
+              <div class="program-title">
+                <h2>Afro Beats</h2>
+              </div>
+              <div class="program-description">
+                <p>
+                  Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                  Consequuntur, consectetur. Placeat et enim quae, ipsa
+                  perspiciatis optio sint eius quaerat?
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="see-more">
+            <a href="#" class="see-more"> SEE THE WHOLE PROGRAM </a>
+          </div>
+          <button class="btn-prog" type="button">Join RG Music Festival</button>
+        </div>
+      </section>
+
+      <section class="third-sec">
+        <div class="third-sec-container">
+          <div class="third-sec-heading">
+            <h2>Performing Artists</h2>
+            <div class="underline2"><hr /></div>
+          </div>
+
+          <div class="flx1"></div>
+        </div>
+
+        <div class="artist-cont">
+          <a href="#">More <i class="fa-solid fa-angle-down"></i></a>
+        </div>
+      </section>
+      <section class="last-sec">
```

Comment:
- [ ] @Enning94 great work in this section, your design would be much better if you reduced the margin and padding between the partners so that they are more centered.